dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -269,6 +269,11 @@ async function handleClientside(
         dc.callback_context.inputs = inputDict;
         dc.callback_context.states_list = state;
         dc.callback_context.states = stateDict;
+        dc.callback_context.outputs_list =
+            Object.prototype.toString.call(outputs) === '[object Object]'
+                ? [outputs]
+                : outputs;
+        dc.callback_context.outputs = output;
```

Comment:
It would be best to make the client side `callback_context` be the same as what's available on the server side.  To achieve this, I think all that's necessary is to add the following here:

```
dc.callback_context.output_list = outputs;
```

Reply:
Ah, indeed! I made a mistake. I will make the necessary changes. Thank you!